packages/transformer/src/IModelTransformer.ts

```diff
@@ -2789,8 +2839,18 @@ export class IModelTransformer extends IModelExportHandler {
         this.context.remapElement(sourceElementId, targetElementId);
       }
     );
-    if (this._csFileProps === undefined || this._csFileProps.length === 0)
-      return;
+    this.exporter.addCustomChanges();
```

Comment:
In my implementation I need to throw error if there are no changes (in source db&custom), do you have suggestions for best way I could achieve it? My first attempt looks something like this:
```
      await this.initialize();
      const isEmpty = this.exporter.sourceDbChanges === undefined || this.exporter.sourceDbChanges.isEmpty;
      if (isEmpty)
          throw xxxx

```

Reply:
That seems appropriate, would be slightly different with the change from isEmpty to hasChanges